releases/latest/mysql-k8s-bundle.yaml

```diff
@@ -0,0 +1,20 @@
+bundle: kubernetes
+name: mysql-k8s-bundle
+type: bundle
+applications:
```

Comment:
Just a note: a PR will be automatically created to order those fields alphabetically as described on https://github.com/canonical/postgresql-bundle/pull/8#discussion_r984853762.

Reply:
good catch! sorted alphabetically in c7c870362d33e40e70c1d2ff64400dc4c6e33b1b